### PR TITLE
Navigation direction prop

### LINKED ISSUE #3720: Navigation direction
**Is your feature request related to a problem? Please describe.**
We are not passing the direction to the Navigation so the dividers lines are wrong for `rtl` scripts as the CSS is not applied properly.

navigation-dir

**Describe the solution you'd like**
Get the direction from the service config file in the Navigation Container and pass it to the `Navigation` component as a prop.

`<Navigation script={script} skipLinkText={skipLinkText} service={service}> dir={dir}`

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [X] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by DenisHdz**
Resolves #3720

**Overall change:** 
Pass direction prop to the Navigation to use the right CSS for `ltr` and `rtl`.

**Code changes:**
- Get `dir` prop from config
- Pass `dir` prop to the `Navigation` component
- Update `Navigation` story to use `variant`
- Bump `psammead-navigation` dependency
- Update snapshots

---

- [X] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [X] This PR requires manual testing

### CODE REVIEW

**Review by ghost — APPROVED** [review]

**Review by AlistairGempf — APPROVED** [review]
Looks good

**Review by dr3 — APPROVED** [review]

**Review by sareh — CHANGES_REQUESTED** [review]
Looks good. However this isn't building on CI since there are duplicate dependencies as a result of pulling in this updated Psammead component. If you run `npm update` and only include the updated `@bbc/psammead-*` packages, that should fix the issue.

**Comment by DenisHdz** [comment]
> Looks good. However this isn't building on CI since there are duplicate dependencies as a result of pulling in this updated Psammead component. If you run `npm update` and only include the updated `@bbc/psammead-*` packages, that should fix the issue.

@sareh I was wondering what was the issue. Running `npm deduped` fixed it. Thank you :)

File: src/app/containers/Navigation/index.jsx [hunk 1]
```diff
@@ -6,17 +6,23 @@ import Navigation, {
...
     return null;
   }
 
   return (
-    <Navigation script={script} skipLinkText={skipLinkText} service={service}>
+    <Navigation
+      script={script}
+      skipLinkText={skipLinkText}
+      service={service}
+      dir={dir}
```
Comment (mykeels) [inline src/app/containers/Navigation/index.jsx:24]: Do we need a default value for `dir`, or is there a test to ensure that the value is always set in the service config?

Comment (DenisHdz) [reply src/app/containers/Navigation/index.jsx:24]: The `dir` prop is not required in the `Navigation` component and by default is set to `ltr`.

**Review by sareh — APPROVED** [review]

**Comment by paruchurisilpa** [comment]
Checked that for `rtl` scripts the navigation styling is fine. Looks good to me..

**Pull request merged**